pkg/operation/botanist/component/kubescheduler/testdata/kube-scheduler.rules.yaml

```diff
@@ -0,0 +1,52 @@
+groups:
```

Comment:
Why is this file commited? 😕

Reply:
This PR was very old and I have some problems with the rebase. Maybe then I missed it.
Sorry.